Atomic/backends/_docker.py

```diff
@@ -97,8 +97,11 @@ def _make_image(self, image, img_struct, deep=False, remote=False):
         img_obj = Image(image, remote=remote)
         img_obj.backend = self
 
+        if img_struct is None:
+            return img_obj
+
         if not remote:
-            img_obj.id = img_struct['Id']
+            img_obj.id = img_struct['Id'] or None
```

Comment:
Explain this syntax?  Would img_struct['Id'] have to be None to get the second option?